### PR TITLE
Feat/mrx select

### PR DESCRIPTION
**Pull request by jjarzbach**
CLOSES https://github.com/monsieur-x/android-app/issues/109

### CODE REVIEW

File: app/src/main/java/io/monsieurx/app/android/view/LobbyActivity.java [hunk 1]
```diff
@@ -116,6 +119,18 @@ public void onClick(View v)
...
+
+            _playerListView.setOnItemClickListener(new AdapterView.OnItemClickListener()
+            {
+                @Override
+                public void onItemClick(AdapterView<?> parent, View view, int position, long id)
+                {
+                    setMonsiuerX(position);
+                    ServiceProvider.getGameService().getCurrentGame().setMonsieurX(_players.get(
+                            position));
+                    ServiceProvider.getGameService().mrx(_players.get(position), null, null);
```
Comment (likesmusik) [inline app/src/main/java/io/monsieurx/app/android/view/LobbyActivity.java]: Can't you do something like:
Player player = _players.get(position);
ServiceProvider.getGameService().mrx(player, null, null);
setMonsiuerX(position);

Is this needed?
ServiceProvider.getGameService().getCurrentGame().setMonsieurX(_players.get(
+                            position));

File: app/src/main/java/io/monsieurx/app/android/view/LobbyActivity.java [hunk 2]
```diff
@@ -166,11 +187,34 @@ private void refreshPlayers()
...
+
+        setMonsiuerX(monsieurXPos);
+    }
+
+    private void setMonsiuerX(int position)
+    {
+        ImageView img;
+        for (int c = 0; c < _playerListView.getCount(); c++)
+        {
+            if (_playerListView == null || _playerListView.getChildAt(c) == null)
```
Comment (nstaeger) [inline app/src/main/java/io/monsieurx/app/android/view/LobbyActivity.java]: `_playerListView == null` can be removed. Would result in Nullpointer in L202

File: app/src/main/java/io/monsieurx/app/android/view/LobbyActivity.java [hunk 3]
```diff
@@ -166,11 +187,34 @@ private void refreshPlayers()
...
+                    R.layout.player_list_item,
+                    R.id.player_list_item_text,
                     _playerNameList);
             _playerListView.setAdapter(adapter);
         }
+
+        setMonsiuerX(monsieurXPos);
+    }
+
+    private void setMonsiuerX(int position)
```
Comment (ManekenT) [inline app/src/main/java/io/monsieurx/app/android/view/LobbyActivity.java]: setMonsiuerX -> setMonsieurX

File: app/src/main/java/io/monsieurx/app/android/view/LobbyActivity.java [hunk 4]
```diff
@@ -153,11 +166,19 @@ private void onSettingsClick(View view)
 
     private void refreshPlayers()
     {
-        List<Player> players = ServiceProvider.getGameService().getCurrentGame().getPlayers();
+        Game game = ServiceProvider.getGameService().getCurrentGame();
```
Comment (nstaeger) [inline app/src/main/java/io/monsieurx/app/android/view/LobbyActivity.java]: die Variable würde ich `currentGame` nennen

Comment (nstaeger) [inline app/src/main/java/io/monsieurx/app/android/view/LobbyActivity.java]: die Variable würde ich `currentGame` nennen

**Pull request merged**